pkg/minikube/registry/registry.go

```diff
@@ -68,11 +68,12 @@ type StatusChecker func() State
 
 // State is the current state of the driver and its dependencies
 type State struct {
-	Installed bool
-	Healthy   bool
-	Error     error
-	Fix       string
-	Doc       string
+	Installed   bool
+	Healthy     bool
+	Error       error
+	Fix         string
+	Doc         string
+	FixRequired bool
```

Comment:
do we need the extra flag? Can't we use healthy, like we are using for the `domcapabilities`?

Reply:
On further investigation the code can use the Healthy flag. Changes made.